controllers/nodehealthcheck_controller.go

```diff
@@ -418,21 +441,21 @@ func (r *NodeHealthCheckReconciler) remediate(node *v1.Node, nhc *remediationv1a
 		return pointer.Duration(1 * time.Second), nil
 	}
 
-	now := metav1.Now()
-	timeoutAt := startedRemediation.Started.Add(*timeout)
+	now := metav1.Time{Time: currentTime()}
```

Comment:
Nit: IIUC this is just a bit of an optimization that's not directly connected to the PR, right ?